docs/tutorials/dockerimages.md

```diff
@@ -254,7 +254,7 @@ First, create a directory and a `Dockerfile`.
     $ touch Dockerfile
 
 If you are using Docker Machine on Windows, you may access your host
-directory by `cd` to `/c/Users/your_user_name`.
+directory by `cd` to `c:/Users/your_user_name`.
```

Comment:
I'm not sure this is correct; depending on the shell that's used (e.g. git bash), the path has to be specified as

`/c/Users/your_user_name` or `C:\Users\your_user_name`

@friism  could you have a look?